### ISSUE TITLE
An option to increase the card sizes to take into account of bleed when printing

### ISSUE THREAD

**Issue by yungspacepup** [mephitrpg/rpg-cards#106]
It'd be great if we could have an option to increase the border of the cards by say 2-4px. This is to take into account bleeding so when printing the physical copies, we can get a more accurate cut, especially when we want to get them double sided.

**Comment by crobi** [comment]
Yes,. the generator needs better settings for border size, bleed margins, and margins between cards.

**Comment by mephitrpg** [comment]
https://github.com/crobi/rpg-cards/commit/15b3d61a82e9b92187df2e048eba48c326ced09c
